packages/effects-core/src/math/shape/vector-feather-renderer.ts

```
@@ -252,14 +319,41 @@ export class VectorFeatherRenderer {
+  * 绘制 Gather Pass。注意Gather Pass不需要对应的Indicator
+  * 在upsample使用几何空间quad时，这个quad可以直接用于gather。
+  */
+  drawGatherPass(
+    renderer: Renderer, 
+    orthoProjection: Matrix4, 
+    featherRadius: number
+  ): void{
+    this.gatherMaterial.setMatrix('uProjection', orthoProjection);
+    this.gatherMaterial.setFloat('uRadius', featherRadius);
+    this.gatherMaterial.setInt('uMeshStart', 0); 
+    var numEdges : number = this.scatterEdgeTexture?.getWidth() || 0;
```

ESLint

[error] Unexpected space before the ':'.

(`@typescript-eslint/type-annotation-spacing`)

---

[error] Expected blank line before this statement.

(padding-line-between-statements)